script.js

```diff
@@ -1,12 +1,82 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
-
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
-
-window.onload = setup;
+//You can edit ALL of the code here
+
+function setup() {
+  const selection = document.getElementById("bar");
+  selection.addEventListener("change", () => {
+    showTech(selection.value);
+  });
+  const img = document.getElementById("logo");
+  img.addEventListener("click", () => {
+    window.location = "./";
+  });
+}

```

Comment:
good use of setup function here. Rather than naming your function `showTech` I would probably name this something like `showPage` - it just makes it clear what the function is doing :)
Also as mentioned before, you can get rid of this img event listener here and just use an `a` tag in your html